scheduler/scheduler.go

```diff
@@ -45,6 +45,10 @@ const (
 var (
 	authProvider = flag.String("mesos_authentication_provider", sasl.ProviderName,
 		fmt.Sprintf("Authentication provider to use, default is SASL that supports mechanisms: %+v", mech.ListSupported()))
+	schedulerHost = flag.String("mesos_scheduler_ip", "0.0.0.0",
+		fmt.Sprintf("TCP4 Address for the Scheduler to bind"))
```

Comment:
hm, I just run it and you are right, it doesn't bring the error, strange as I thought it should, ok, sorry for the noise